src/ducks/Alert/components/AlertModalContent/StepsContent/NotificationAndPrivacy/ChannelsSelector/ChannelsSelector.js

```diff
@@ -0,0 +1,153 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import { useField } from 'formik'
+import Message from '@santiment-network/ui/Message'
+import EmailToggle from './channels/EmailToggle/EmailToggle'
+import TelegramToggle from './channels/TelegramToggle/TelegramToggle'
+import PushToggle from './channels/PushToggle/PushToggle'
+import WebhookToggle from './channels/WebhookToggle/WebhookToggle'
+import { useUserSettings } from '../../../../../../../stores/user/settings'
+import { useUser } from '../../../../../../../stores/user'
+import { getSanSonarSW } from '../../../../../../../pages/Account/SettingsSonarWebPushNotifications'
+import { getChannelsTitles } from '../../../../../utils'
+import styles from './ChannelsSelector.module.scss'
+
+const ChannelsSelector = () => {
+  const [, { value }, { setValue }] = useField('settings.channel')
+  const [isPushDisabled, setIsPushDisabled] = useState(false)
+  const {
+    user: { email }
```

Comment:
Since `user` can be `null`, we have to handle this case before destructuring. 
_(It's implied that the settings page only can be opened if the `user` has a value, but in case if we want to reuse this component somewhere else it might not be true)_